core/views.py

```diff
@@ -85,3 +87,18 @@ def form_invalid(self, form):
             messages.error(self.request, error.as_data()[0].message)
 
         return self.get()
+
+
+class FeedbackView(LoginRequiredMixin, FormView):
+    template_name = 'feedback.html'
+    form_class = FeedbackForm
+
+    def form_valid(self, form):
+        message = mail.EmailMessage(form.cleaned_data['subject'],
+                                    form.cleaned_data['message'],
+                                    settings.NO_REPLY_EMAIL,
+                                    [settings.FEEDBACK_EMAIL],
+                                    headers={'Reply-To': self.request.user.email})
+        message.send()
+        messages.success(self.request, _('Thank you for your feedback! We\'ll get in touch as soon as possible.'))
+        return HttpResponseRedirect('/events/')
```

Comment:
Here you can use the Django's reverse module. This way if we change this URL it won't raise errors.